R/utilities-population.R

```diff
@@ -9,7 +9,30 @@
 #' @export
 loadPopulation <- function(csvPopulationFile) {
   validateIsString(csvPopulationFile)
-  populationImporter <- getNetTask("PopulationTask")
-  population <- rClr::clrCall(populationImporter, "ImportPopulation", csvPopulationFile)
+  populationTask <- getNetTask("PopulationTask")
+  population <- rClr::clrCall(populationTask, "ImportPopulation", csvPopulationFile)
   Population$new(population)
 }
+
+#' Loads a population from the \code{csvPopulationFile} and split the loaded population according to
+#' \code{numberOfCores}.
+#' @param csvPopulationFile Full path of csv population file to split.
+#' @param numberOfCores Number of cores used for parallelisation computing. The population will be split accross all cores.
+#' @param outputFolder Folder where all split files will be created
+#' @param outputFileName File names will be constructed using this parameter concatenated with the core index.
+#' @return A tring vector containing the full path of the population files created. Note that there might be less files than cores
+#'
+#' @examples
+#' csvPath <- system.file("extdata", "pop.csv", package = "ospsuite")
+#'
+#' # Split the population in up to 3 files, saved in the temp folder
+#' splitFiles <- splitPopulationFile(csvPath, 3, tempdir(), "PopFile")
+#' @export
+splitPopulationFile <- function(csvPopulationFile, numberOfCores, outputFolder, outputFileName) {

```

Comment:
@abdullahhamadeh This method is now available for you to use. It will split the csv files in `numberOfCores` subfiles (more precisely up to the number) and save the file in the output folder

for example

```
splitPopulationFile(csv, 5, "C:\temp", "popFile") 
```
will create 
C:\temp\popFile_1
C:\temp\popFile_2
etc..

The created files are returned from the function so you know where they are located and how many were created. It can be less than the number of cores. For example if you have 960 Individuals and 50 cores we will end up with 20 Individuals per cores and thus the last 2 cores will have nothing todo.